src/gui/gtkentry.c

```diff
@@ -210,19 +211,22 @@ const dt_gtkentry_completion_spec *dt_gtkentry_get_default_path_compl_list()
           { "ELEVATION", N_("$(ELEVATION) - elevation") },
           { "STARS", N_("$(STARS) - star rating as number (-1 for rejected)") },
           { "RATING_ICONS", N_("$(RATING_ICONS) - star/reject rating in icon form") },
-          { "LABELS", N_("$(LABELS) - colorlabels") },
+          { "LABELS", N_("$(LABELS) - color labels at text") },
```

Comment:
as text ?

Reply:
Cannot test right now but yes, IIRC the output is using color names, for example `red, green`.

Reply:
I meant "as" instead of "at" ... :)

Reply:
And you know what I did read "as" :)